packages/headless/src/features/insight-search/insight-search-request.ts

```diff
@@ -55,6 +56,10 @@ export const buildInsightSearchRequest = (
       sortCriteria: state.sortCriteria,
     }),
     tab: state.configuration.analytics.originLevel2,
+    filterField: state.folding?.fields.collection,
+    childField: state.folding?.fields.parent,
+    parentField: state.folding?.fields.child,
+    filterFieldRange: 100,
```

Comment:
This should not always be 100. The value of this parameter should be 100 only in the `loadCollection` action.
```suggestion
    filterFieldRange: state.folding.filterFieldRange,
```